lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/migration/MigrationType.json

```diff
@@ -30,8 +30,8 @@
         "DOCKER_REPOSITORY_NAME",
         "DOCKER_COMMIT",
         "TEAM",
-        "MEMBERSHIP_INVITATION_SUBMISSION",
-        "MEMBERSHIP_REQUEST_SUBMISSION",
+        "MEMBERSHIP_INVITATION",
+        "MEMBERSHIP_REQUEST",
```

Comment:
Check with John and/or Xa:  Are we free to change these names, or will it trip up the migrator.